### ISSUE TITLE
stuck at Awaiting Login

### ISSUE THREAD

**Issue by Hecticism** [ee3213/lol-leveling-bot#2]

**Comment by Krevzy** [comment]
Yes, me too:(

**Comment by AmauryVa** [comment]
Yes, because your game is not in english. I currently work on the project. I will release the new code and documentation on my repo because i can't create a branch on this repo.

If your game is in english, it's because you launch the script on the wrong place.

**Comment by livieramaury** [comment]
My Repo => https://github.com/Krach64/lol-bot-leveling-free with corrected issues
